alerting/src/main/kotlin/org/opensearch/alerting/action/GetAlertsRequest.kt

```diff
@@ -17,25 +17,29 @@ class GetAlertsRequest : ActionRequest {
     val severityLevel: String
     val alertState: String
     val monitorId: String?
+    val alertIndex: String?
 
     constructor(
         table: Table,
         severityLevel: String,
         alertState: String,
-        monitorId: String?
+        monitorId: String?,
+        alertIndex: String?,
```

Comment:
Do we need to add the version check for backward compatibility?

Reply:
created a task to track this.

Reply:
this question applies to changes that we have been making in the overall Datasources purview. WIll revisit during refactoring models to common utils